package.json

```diff
@@ -19,7 +19,7 @@
     "test:without-system-tests": "lerna run test --ignore system-tests --concurrency 1",
     "test:system-tests": "lerna run test --scope system-tests --concurrency 1",
     "coverage:system-tests": "lerna run coverage --scope system-tests --concurrency 1",
-    "vscode:package": "lerna run vscode:package --concurrency 1",
+    "vscode:package": "lerna exec --scope @salesforce/salesforcedx-* -- npm prune --production && lerna run vscode:package --concurrency 1",
```

Comment:
The stuff in @salesforce/salesforcedx-* are the plain npm modules. These are depended on by the actual extensions.